src/game/logic/object/objecttypes.cpp

```diff
@@ -44,3 +48,80 @@ void ObjectTypes::Xfer_Snapshot(Xfer *xfer)
         }
     }
 }
+
+Utf8String ObjectTypes::Get_Nth_In_List(unsigned int i) const
+{
+    if (i > Get_List_Count()) {
+        return Utf8String::s_emptyString;
+    } else {
+        return m_objectTypes[i];
```

Comment:
This is undefined behavior when `i == Get_List_Count()`.

Reply:
Fixed.